### DISCUSSION TITLE
I cant unblock my blocked element

### DISCUSSION THREAD

**Discussion by Alidani123** [uBlockOrigin/uAssets#19989] [General]
### Prerequisites

- [X] I verified that this is not a filter list issue. Report any issues with filter lists or broken website functionality in the [uAssets issue tracker](https://github.com/uBlockOrigin/uAssets/issues).
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uBlock-issues/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] The issue is not present after disabling uBO in the browser.
- [X] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I am reporting is not normal behavior.

### I tried to reproduce the issue when...

- [X] uBO is the only extension.
- [X] using a new, unmodified browser profile.

### Description

i cant unblock  my blocked element

### A specific URL where the issue occurs.

i dont understand the question

### Steps to Reproduce

go in to ublock 

unblock the blocke element

### Expected behavior

i accidently blocked my school port and the job to the school

### Actual behavior

i accidently blocked my school port and the job to the school

### Configuration

<details>

```yaml
uBlock Origin: 1.52.2
Chromium: 116
filterset (summary):
 network: 106672
 cosmetic: 42337
 scriptlet: 18978
 html: 0
listset (total-discarded, last-updated):
 removed:
  ublock-quick-fixes: null
 added:
  https://drive.google.com/uc?export=download&id=1Ito3Ta23mJBmHaDw2m6sBWtsRs1sTOa8: 29-0, 1h.10m
 default:
  user-filters: 12-0, never
  SWE-1: 4667-0, 4d.22m
  easylist: 71318-19, 4d.23m
  easyprivacy: 33195-68, 4d.22m
  plowe-0: 3720-1065, 4d.23m
  ublock-badware: 7645-138, 4d.23m
  ublock-filters: 34904-142, 4d.23m
  ublock-privacy: 557-6, 2d.23h.36m
  ublock-unbreak: 2127-31, 4d.23m
  urlhaus-1: 11425-0, 1h.10m
filterset (user): [array of 12 redacted]
switchRuleset:
 added: [array of 2 redacted]
userSettings:
 collapseBlocked: false
 contextMenuEnabled: false
 showIconBadge: false
hiddenSettings: [none]
supportStats:
 allReadyAfter: 6243 ms (selfie)
 maxAssetCacheWait: 804 ms
```
</details>

**Comment by Alidani123** [comment]
how do i unblock in uBlock side

**Reply by mapx-** [reply]
open uBO dashboard, "my filters" tab, remove your wrong filters, confirm it
